cmd/controller/app/controller.go

```diff
@@ -120,8 +124,8 @@ func buildControllerContext(opts *options.ControllerOptions) (*controller.Contex
 	// If it is specified, all operations relating to ClusterIssuer resources
 	// should be disabled and thus we don't need to also create factories for
 	// the --cluster-resource-namespace.
-	sharedInformerFactory := informers.NewFilteredSharedInformerFactory(intcl, time.Second*30, opts.Namespace, nil)
-	kubeSharedInformerFactory := kubeinformers.NewFilteredSharedInformerFactory(cl, time.Second*30, opts.Namespace, nil)
+	sharedInformerFactory := informers.NewFilteredSharedInformerFactory(intcl, time.Second*30, defaultNamespace, nil)
```

Comment:
We can stop using the filtered variants of these functions and therefore remove the `defaultNamespace` const altogether. (`NewSharedInformerFactory(intcl, time.Second*30)`)